classes/services/PKPSchemaService.inc.php

```diff
@@ -377,7 +377,7 @@ public function addPropValidationRules($rules, $ruleKey, $propSchema)
                 }
                 break;
             case 'object':
-                foreach ($propSchema->properties as $subPropName => $subPropSchema) {
+                foreach ($propSchema->properties ?? [] as $subPropName => $subPropSchema) {
```

Comment:
There's a lot of warnings related to this.

This object is built from a `json_decode()` over the JSON schema, and [some "objects" in the schema are missing the `properties`](https://github.com/pkp/pkp-lib/blob/main/schemas/decision.json#L23-L25), I didn't dig deeper than that (for me looks like a way to declare a generic object without fixed properties).

Another way to fix this would be to force declaring the properties in the schema file, so I could add a `"properties": []`.

Reply:
Moved to https://github.com/pkp/pkp-lib/issues/7995